### PR TITLE
Feature/ROC-3223-Smoke test legal front end

### PR DESCRIPTION
**Pull request by vianvi**
**Before creating a pull request make sure that:**

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: docker-compose.local.yml [hunk 1]
```diff
@@ -7,8 +7,9 @@ services:
         - http_proxy=
         - https_proxy=
         - no_proxy=
-        - SMOKE_TEST_USERNAME=test1@test.com
+        - SMOKE_TEST_USERNAME=testcitizensmoke@test.com
```
Comment (timja-kainos) [inline docker-compose.local.yml]: please only use real email addresses or override them from vault

Comment (vianvi) [reply docker-compose.local.yml]: updated to real email addresses

File: docker-compose.local.yml [hunk 2]
```diff
@@ -7,8 +7,9 @@ services:
         - http_proxy=
         - https_proxy=
         - no_proxy=
-        - SMOKE_TEST_USERNAME=test1@test.com
+        - SMOKE_TEST_USERNAME=civilmoneyclaims+citizensmoke@gmail.com
```
Comment (geek-soft) [inline docker-compose.local.yml]: If we have to have two users, shall we name that PR a `SMOKE_TEST_CITIZEN_USERNAME`? The same way we would have `SMOKE_TEST_SOLICITOR_USERNAME` and `SMOKE_TEST_PASSWORD` (shared between both users)

REQ

File: src/bootstrap/bootstrap.ts [hunk 3]
```diff
@@ -95,7 +95,8 @@ module.exports = async function (done: () => void) {
     }
 
     await Promise.all(healthChecks)
-    await createSmokeTestsUserIfDoesntExist()
+    await createSmokeTestsUserIfDoesntExist(process.env.SMOKE_TEST_USERNAME,'cmc-private-beta',process.env.SMOKE_TEST_PASSWORD)
```
Comment (geek-soft) [inline src/bootstrap/bootstrap.ts]: Can you please add space between call parameters?

File: src/tests/legal/home/steps/user.js [hunk 4]
```diff
@@ -19,18 +19,19 @@ module.exports = {
     claimantAddressPage = require('../../claim/pages/claimant-address')
     claimantAddPage = require('../../claim/pages/claimant-add')
   },
-  loginUser (userEmail) {
+  loginUser (userEmail, password) {
```
Comment (geek-soft) [inline src/tests/legal/home/steps/user.js]: You can also do `loginUser (userEmail, password = DEFAULT_PASSWORD)`. Both ways are correct, you do not need to change it.

File: src/tests/legal/home/steps/user.js [hunk 5]
```diff
@@ -19,18 +19,19 @@ module.exports = {
...
-    loginPage.login(userEmail, DEFAULT_PASSWORD)
+    loginPage.login(userEmail, password || DEFAULT_PASSWORD)
   },
   startClaim () {
     startClaimPage.open()
     startClaimPage.startClaim()
   },
-  loginAndStartClaim (userEmail) {
-    this.loginUser(userEmail)
+  loginAndStartClaim (userEmail, password) {
```
Comment (geek-soft) [inline src/tests/legal/home/steps/user.js]: In the future I would encourage removing this method. Methods that have 'and' in a name typically mean that method is doing too much. The API should be modular and reusable, that breaks this rule.

Something for the future.

File: docker-compose.local.yml (hunk 2 above)
Comment (geek-soft) [reply docker-compose.local.yml]: I put citizen username under `secret/dev/cmc/smoke-tests/citizen-username` key in vault.

File: src/tests/legal/home/steps/user.js (hunk 4 above)
Comment (vianvi) [reply src/tests/legal/home/steps/user.js]: ok , thank you.

File: src/tests/legal/home/steps/user.js (hunk 5 above)
Comment (vianvi) [reply src/tests/legal/home/steps/user.js]: sure will do it in future

File: docker-compose.local.yml (hunk 2 above)
Comment (vianvi) [reply docker-compose.local.yml]: variable name updated

File: src/bootstrap/bootstrap.ts (hunk 3 above)
Comment (vianvi) [reply src/bootstrap/bootstrap.ts]: done

**Pull request merged**